### ISSUE TITLE
Es6 Is not supporet

### ISSUE THREAD

**Issue by timsar2** [jalaali/moment-jalaali#245]
Es6 is not supported. this package is dependent on CommonJS or AMD.
that can cause optimization bailouts when bundling modern javascript.

**Comment by behrang** [comment]
You're right, but what's your proposal? I think this project is getting retired and it's better to not make a big change like this.
